### PR TITLE
[JDK17&21 on Release Test] LPD-29624 Use jdk21_zulu and jdk17_open for release tester test batches

### PR DESCRIPTION
**Pull request by julschong**
resubmitted from https://github.com/liferay-core-infra/liferay-portal/pull/7316

@tinatian @vicnate5 
Since Victor is on PTO, we can wait for him to be back to review this change.

ci team helped running the entire tester with my changes here: https://testray.liferay.com/#/project/34754/routines/730483/build/70800659

5% incomplete tests is mostly due to unrelated ci testing issue

image

### CODE REVIEW

File: test.properties [hunk 1]
```diff
@@ -7872,7 +7872,7 @@ test.batch.run.property.query[functional-orcllinux9-tomcat90-mysql57-jdk8]=\
         functional-smoke-bundle-wildfly230-mysql57-jdk21_zulu,\
         functional-smoke-bundle-wildfly261-mysql57-jdk21_zulu,\
         patching-tool-mysql57-jdk8,\
-        playwright-js-bundle-tomcat90-mysql57-jdk8
+        playwright-js-tomcat90-mysql57-jdk21_zulu
```
Comment (vicnate5) [inline test.properties:7875]: @julschong Why is the `-bundle-` naming convention changed here?

Comment (julschong) [reply test.properties:7875]: @vicnate5 This is the why it was changed https://github.com/liferay-core-infra/liferay-portal/pull/7316#issuecomment-2269922806

did I misunderstand anything?

Comment (vicnate5) [reply test.properties:7875]: I meant that we can use the non-bundle one to test out the change beforehand.

The final version of the batch for the release tester should be `playwright-js-bundle-tomcat90-mysql57-jdk21_zulu` to follow the naming convention.

Comment (julschong) [reply test.properties:7875]: @vicnate5 Okay, let me change it back

File: build-test-batch.xml [hunk 2]
```diff
@@ -8003,6 +8003,10 @@ ${output.content}</echo>
 		/>
 	</target>
 
+	<target name="playwright-js-bundle-tomcat90-mysql57-jdk21_zulu">
+		<run-playwright-js-test />
```
Comment (julschong) [inline build-test-batch.xml:8007]: @vicnate5 please see this change

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
see https://github.com/brianchandotcom/liferay-portal/pull/154054

**Pull request closed without merging**